metrics/web/test/dashboard/presentation/widgets/metrics_table_title_header_test.dart

```diff
@@ -79,38 +54,92 @@ void main() {
     );
 
     testWidgets(
-      "displays the performance header",
+      "displays the TooltipTitle with the last builds title, last builds description and info src",
       (tester) async {
-        await tester.pumpWidget(const _MetricsTableTitleHeaderTestbed());
-
-        expect(find.text(DashboardStrings.performance), findsOneWidget);
+        await mockNetworkImagesFor(() {
+          return tester.pumpWidget(const _MetricsTableTitleHeaderTestbed());
+        });
+
+        final finder =
+            find.widgetWithText(TooltipTitle, DashboardStrings.lastBuilds);
+        final tooltipTitle = tester.widget<TooltipTitle>(finder);
+        final tooltip = tooltipTitle.tooltip;
+
+        expect(finder, findsOneWidget);
+        expect(tooltipTitle.src, equals(src));
+        expect(tooltip, equals(DashboardStrings.lastBuildsDescription));
```

Comment:
Should we split this test into two separate tests? Also, it seems like `expect(finder, findsOneWidget);` is redundant here.